ovos_bus_client/session.py

```diff
@@ -172,23 +173,49 @@ def from_message(message=None):
                     sess.lang = lang
             else:
                 sess = Session(lang=lang)
+                LOG.debug(f"Created new session: {sess}")
         else:
             # new session
+            LOG.warning(f"No message found, creating a new session")
             sess = Session()
```

Comment:
line 180 is bug, a new session should not be created, default session should be used here